### DISCUSSION TITLE
Missing Icons

### DISCUSSION THREAD

**Discussion by johnnyneptunenyc** [elrumo/macOS_Big_Sur_icons_replacements#1293] [Help with the website]
Anyone notice some icons have gone missing? When I try to download this one (https://media.macosicons.com/parse/files/macOSicons/445e5fbe42539be75ba48a36dd7103a9_Arturia_Software_Center.icns) all I get is a blank page. Is this a bug?

**Comment by cake03** [comment]
https://github.com/elrumo/macOS_Big_Sur_icons_replacements/issues/1291

**Reply by OMGzilla-git** [reply]
Same here. The last few of mine that were approved now show as blank, and one that's been waiting on approval for... gosh.. a couple months now? Also blank.
